spec/command_line_ui_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "../lib/command_line_ui"
+require "rspec"
+
+class CommandLineUITest
+  describe CommandLineUI do
+    it "" do
+      
+    end
+  end
+end
```

Comment:
which file? "command_line_ui?"  I'd set it up in prep.  It's now needed as I instantiate a CommanLineUI object